src/Data/Arity/Linear/Internal.hs

```diff
@@ -19,3 +62,26 @@ type family Arity b f where
           ':<>: 'ShowType b
```

Comment:
But now, `Arity` seems to work on non-linear functions (because of `%p ->`), so maybe we should change the error message?

Reply:
This, as well, is debugging code which was left over…